src/grocker/builders.py

```diff
@@ -304,23 +296,25 @@ def inspect_stream(stream):
 
 def docker_pull_image(docker_client, name):
     logger.info('Pulling %s...', name)
-
-    stream = docker_client.pull(name, stream=True)
-    inspect_stream(stream)
-    return [image for image in docker_client.images() if name in (image['RepoTags'] or [])]
+    try:
+        stream = docker_client.images.pull(name, stream=True)
```

Comment:
I think we could use the low-level API here, with `docker_client.api.pull` (and `decode=True`) and provide a nice `--verbose` option to decide what logs we want to show.